docs/content.md

```diff
@@ -64,11 +64,10 @@ Login.gov’s voice is&hellip;
 
 |Correct term or phrase   	|Common errors   	|More information   	|
 |---	|---	|---	|
-|login.gov   	|Login; Login.gov; LOGIN.gov; LOGIN   	|Use all lowercase and include the “.gov”. When at the beginning of a sentence, capitalize the first L.   	|
```

Comment:
Would it be helpful to replace this with the inverted guidance? i.e. "login.gov" lowercase as a common error.

Reply:
This is something I did think about. Login.gov is a proper noun, so we do not need to call the inverted out. It defaults to grammatical rules rather than branding rules.